File: backend/internal/models/sms.go
package models

// SmsLog represents a row in the sms_logs table.
type SmsLog struct {
	ID      int    `json:"id"`
	Phone   string `json:"phone"`
	Message string `json:"message"`
	Status  string `json:"status"`
	SentAt  string `json:"sent_at"`
}

File: frontend/lib/models/sms.dart
// lib/models/sms.dart — SMS message data model.
//
// Fields:
//   int id, int farmId, String phoneNumber, String message, String status, String sentAt
//
// Responsibilities:
//   - Define an SmsMessage class with a fromJson() factory constructor.
//   - JSON keys match the Go backend GET /api/sms response.
//
// Used by:
//   services/sms_service.dart — deserializes API response.
//   screens/sms/ — displays SMS history list.
class SmsMessage {}